sandboxnu/graduatenu
# Feat nest backend

# Description

A version 2 of the GraduateNU API in TypeScript and Nestjs. 

Closes [Try out nestjs backend](https://trello.com/c/4mJbmpC1/361-try-out-nextjs-backend)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Postman testing
  - Run `yarn install` to fetch dependencies
  - Ensure a community edition of PostgreSQL is running
  - Run `yarn run start:dev` to start up the server
  - Test the server through postman


# Checklist:
